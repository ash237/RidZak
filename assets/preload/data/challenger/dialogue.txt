right:Listen RidZak! We got a big problem here!
switch:2
right:There is some kid running around in MY city, named "boyfriend".
right:GET HIM OUT! He is bad news for business!
right:I don't want him to take away any attention from the club!
switch:3
right:But I heard he is pretty good at what he does, so if you can't beat him, use this. 
switch:99
left:I wont be needing it, trust me. I can finish him in one round.
right:If you say so... now go break a leg.
switch:5
left:...
nobg:fuckyou
ridzak:Word on the street's that you're a pretty good rapper,
ridzak:I would like to test that.
bf:Beep.
ridzak:Give me your best, Boyfriend! Don't hold back. 
