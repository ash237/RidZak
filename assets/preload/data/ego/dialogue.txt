ridzak:I didn't want to use this!
ridzak:But I am left with no other choice...
putthefuckingneedlein:lol
ridzakego:I am sorry, Boyfriend...