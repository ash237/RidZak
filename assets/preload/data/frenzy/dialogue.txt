left:hey!
left:Thank you for checking out our mod!
left:We have 2 bonus songs left for you to check out.
left:Though they're not related to RidZak plus Cybbr.
left:Enjoy!
nobg:AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
rian: Aha! found you!
bf: Beep?
rian: I saw your skills last night. You're CRAZY talented!
bf: Beep bep!
bf: Bee-beep boop?
rian: I guess it was obvious, huh?
bf: Beep.
rian: Well, alright then! a rap battle it is!