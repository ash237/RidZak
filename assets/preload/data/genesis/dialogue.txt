cybbr:Pretty impressive! Lets try another!