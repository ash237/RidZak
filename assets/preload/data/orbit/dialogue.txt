cybbr:Hey you two look like you know a lot about music! 
cybbr:How about you guys try to go against me and my DJ skills??
bf:Beep.