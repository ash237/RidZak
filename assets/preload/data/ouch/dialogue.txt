ridzak: Pretty good for a kid. But you are in my city, and I will win. 
gf: Pretty sure he just beat you on the first song. 
ridzak: I was just only warming up.
gf:Yeah, warming up...