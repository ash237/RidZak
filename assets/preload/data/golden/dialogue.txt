cybbr:WOW alright. I got one trick up my sleeve.
cybbr:Lets see if you can keep up with this!